app/assets/javascripts/discourse/components/pwa-install-banner.js.es6

```diff
@@ -0,0 +1,53 @@
+import {
+  default as computed,
+  on
+} from "ember-addons/ember-computed-decorators";
+
+const userDismissedPromptKey = "dismissed-pwa-install-banner";
+
+export default Ember.Component.extend({
+  deferredInstallPromptEvent: null,
+
+  @on("init")
+  _registerListener() {
+    var that = this;
```

Comment:
why do you need that here ?

Reply:
Just so I can set on line 18. Is there a better way?

Reply:
```
  window.addEventListener("beforeinstallprompt", e => {
      // Prevent Chrome 76+ from automatically showing the prompt
      e.preventDefault();
      // Stash the event so it can be triggered later
      this.set("deferredInstallPromptEvent", e);
    });
  }
```

Should just work as your are using fat arrow. It doesn't?